packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+# 48.23.0 - July 2, 2020
+
+## Added
+
+#### salesforcedx-vscode-apex
+
+- Added 8 new Apex code snippets ([PR #2262](https://github.com/forcedotcom/salesforcedx-vscode/pull/2262), [Issue #2261](https://github.com/forcedotcom/salesforcedx-vscode/issues/2261)) - Contribution by [lukecotter](https://github.com/lukecotter)
+
+#### salesforcedx-vscode-core
+
+- Added tooltips to the org picker and org status bar items ([PR #2272](https://github.com/forcedotcom/salesforcedx-vscode/pull/2272))
+
+#### salesforcedx-vscode-lwc
+
+- Added `SFDX: Stop Watching All Lightning Web Component Tests` command ([PR #2298](https://github.com/forcedotcom/salesforcedx-vscode/pull/2298))
+
```

Comment:
@brpowell , remove Added before each line (it's in the Added section) and rephrase as:
Eight new Apex code snippets
Tooltips for Org Picker and Status Bar items
New `SFDX: Stop Watching All Lightning Web Component Tests` command